src/ContractInteract/OSTPrime.js

```diff
@@ -1,9 +1,27 @@
+/**
+ * @typedef {Object} OSTPrimeSetupConfig
+ *
+ * @property {string} deployer Address to be used to send deployment transactions.
+ * @property {string} chainOwner Address that holds all the base token funds
+ *                    for initializing the OSTPrime contract.
+ * @property {string} valueToken Address of EIP20 Token on Origin chain.
```

Comment:
Do we still use "value token" and "utility token" as concepts? @benjaminbollen @deepesh-kn ?

Reply:
Here the value token is referred to as branded token or ERC20 token from the origin chain.
As per the concepts, we are moving the tokens from one chain to another chain. But in contracts, we have the terms like value token and utility token.